Update go example to reflect our latest improvement for `@now/go` (#777)
zeit/docs: pages/docs/v2/deployments/official-builders/go-now-go.mdx

}
```

> Notice that the API is cohesive with the `net/http` Go API. Your main entrypoint function **must be** an exported function.

Reviewer: What happens if there are 2 exported functions in the entrypoint?

Author: @styfle The second one will be ignore.

Reviewer: @sophearak Maybe we should document this behavior.

What do you think @msweeneydev @timothyis ?

Author: @styfle General rule of thumb, 1 entrypoint, 1 go src file, 1 export function that implement `net/http` interface.

Reviewer 2: Definitely should note it. Let's do that in another PR.

### Entrypoint

> The entrypoint file must be a `.go` source file with an exported function inside implementing the `net/http` API.

Reviewer: ```suggestion
The entrypoint file must be a `.go` source file containing an exported function that implements the `net/http` API.
```

> <Note>

Reviewer: You need to import `Note`

Author: Ahhh, right :/ I was copy and paste from my note. usually in Go, it does for me 😅
